src/lizaalert/courses/admin.py

```diff
@@ -47,6 +49,40 @@ class ChapterInline(admin.TabularInline):
     extra = 0
 
 
+@admin.register(Cohort)
+class CohortAdmin(admin.ModelAdmin):
+    """
+    Админка когорты.
+
+    При созданиии объекта исключается поле cohort_number.
+    """
+
+    model = Cohort
+    extra = 1
+    list_display = (
+        "course_title",
+        "start_date",
+        "end_date",
+        "teacher",
+        "created_at",
+        "updated_at",
+    )
+    ordering = ("-updated_at",)
+
+    def get_fields(self, request, obj=None):
+        fields = super().get_fields(request, obj)
+        return fields
+
+    def get_queryset(self, request):
+        qs = super().get_queryset(request).select_related("course").annotate(course_title=F("course__title"))
+        return qs
+
+    def course_title(self, obj):
+        return obj.course_title
```

Comment:
здесь не нужна аннотация


```suggestion
    def get_queryset(self, request):
        qs = super().get_queryset(request).select_related("course")
        return qs

    def course_title(self, obj):
        return obj.course.title
```

Reply:
Давайте сделаем так: 
https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_select_related

с @Di0niz согласовано